hack/lib/serverless.bash

```diff
@@ -103,6 +106,7 @@ function deploy_knativeserving_v1alpha1_1.3.0 {
   logger.info 'Deploy Knative Serving'
 
   # Wait for the CRD to appear
+  # TODO: use full path name to ensure we're checking the right thing. aka knativeservings.operator.knative.dev VS knativeservings.serving.knative.dev
   timeout 900 "[[ \$(oc get crd | grep -c knativeservings) -eq 0 ]]" || return 6
```

Comment:
I think we need to change this `grep -c knativeservings` to `knativeservings.operator.knative.dev`
Otherwise, I am not sure from what group we would get `knativeservings`.
There's probably a rule there, but I would rather use the long name here. ...unless there's a proper reason.

cc @jcrossley3

Reply:
@aliok technically, it shouldn't matter. either one should work. but since the function has `_latest` in its name, i suppose it should use the fqn to be sure.

Reply:
Ok, following the minimal disruption principle, I won't do anything around this